src/client/components/media-video.vue

```diff
@@ -50,6 +50,29 @@ export default defineComponent({
 	created() {
 		this.hide = (this.$store.state.nsfw === 'force') ? true : this.video.isSensitive && (this.$store.state.nsfw !== 'ignore');
 	},
+	methods: {
+		onPlay() {
+			if (this.dp === null) {
+				// Only for creating new
+				this.dp = new DPlayer({
+					// Configure parameters
+					container: document.getElementById(this.video.id),
```

Comment:
If multiple same video links exists in one page (e.g. renote), this conflicts with other same video link.

Reply:
Reproduced. Figuring out how to resolve it. 😶

Reply:
Should be fixed now. Passing `$event` into the trigger function and use `event.target` to get the clicked element. 😃